snap/snapcraft.yaml

```diff
@@ -0,0 +1,60 @@
+name: matrix-appservice-slack
+base: core18
+version: git 
+summary: Node.js Matrix bridge for Slack
+description: |
+  A bridge between Matrix and Slack. Currently the bridge is in Beta.
```

Comment:
This is now out of date. Use the line from the README

Reply:
OK, revised commit incoming.